main.tf

```diff
@@ -0,0 +1,17 @@
+# The module configures basic monitoring configuration
+
```

Comment:
Please, live an empty line after the existing comment. Before data add "Existing infrastructure", then an empty line. Before resources Add a one-line comment "New infrastructure".
It's just a convention the way we structure modules.